website/docs/reference/project-collaboration-mode.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Project collaboration mode
+---
+
+:::info Availability
+
+The change requests feature is an enterprise-only feature that was introduced in **Unleash 4.22.0**.
+
+:::
+
+Unleash project collaboration mode is a feature that allows to specify who can submit [change requests](change-requests.md). Currently, there are two modes:
+- **open** where everyone can submit change requests. This is a default setting for the existing and new Unleash projects.
+- **protected** where [admins, project members and owners](rbac.md#standard-roles) can submit change requests. Users who are not added to projects will not be able to submit change requests.
+
+## Project collaboration mode setting
+
+Project collaboration mode is controlled from the "Create Project" and "Edit Project" screens.
+
+![Create project with collaboration mode](/img/collaboration-mode.png)
+
+When project collaboration mode is protected all users who are not project members or admins will have all the
+change request operations disabled.
+
+## Enabling protected mode for an existing project
+
+To change the project collaboration mode for an existing project you have to edit the project.
```

Comment:
This feels like it's veering into how-to territory, but maybe we can rephrase it a bit to keep it as a reference guide.

Also, what happens if you change a project's collaboration mode after a viewer submits a CR? Does it stay? And what about editing it?

```suggestion
## Project collaboration mode setting

You can set a project's collaboration mode when you create a project and at any point after creation.

![Project creation form with a collaboration mode field and corresponding explanation.](/img/collaboration-mode.png)

### Change requests 

When you change a project's collaboration mode from open to protected, users who do not have the right permissions will lose the ability to create new change requests. 

However, existing change requests created by these users will not be deleted. Any users with open change requests will still be able to edit the change request, regardless of their current permissions in the project.
```

Reply:
So when you make the project protected then the permission checker kicks in and you can only cancel your change request from this moment.

Reply:
So you can cancel or have it approved & applied. It cannot be updated?

Reply:
Correct. I updated the text in docs to reflect it.